### PR TITLE
[#358] Reworking PVA time limit session card

### PR DESCRIPTION
**Pull request by eubiera**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Introduced a feature that was causing an hour delay in our tests. This PR addresses that. 

- *(What is the solution, why is this the solution)*
We are keep track of timestamps instead of using timers. 

- *(Which team do you work for, does your team own the maintenance of this component?)*
We are the Orchid Chatbot Team.

- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*
N/A.

## Related issue(s)
[https://github.com/department-of-veterans-affairs/va-virtual-agent/issues/358]

- department-of-veterans-affairs/va.gov-team#0000

## Testing done

- *Describe what the old behavior was prior to the change*
Our behavior hasn't changed, however our tests are faster. 

- *Describe the steps required to verify your changes are working as expected*
Run the test suite. 

## Screenshots
N/A.

## What areas of the site does it impact?
VA Chatbot.

## Acceptance criteria

- [ x ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ x ]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
N/A.

- [x]  Documentation has been updated (link to documentation)
N/A.

- [ x ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
N/A
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
N/A
- [x]  I added a screenshot of the developed feature
N/A
- [x]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed
N/A

### CODE REVIEW

**Review by adamwhitlock1 — APPROVED** [review]
Great idea as a different approach towards the time checking! Thanks for getting this resolved.

**Review by helene-mccarron — APPROVED** [review]
LGTM

File: src/applications/virtual-agent/components/chatbox/Chatbox.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
 } from './loadingStatus';
 import { storeUtterances, LOGGED_IN_FLOW, IN_AUTH_EXP } from './utils';
 
-const ONE_MINUTE_IN_MS = 60_000;
+// const ONE_MINUTE_IN_MS = 60_000;
```
Comment (ksk385) [inline src/applications/virtual-agent/components/chatbox/Chatbox.jsx:18]: Please delete

File: src/applications/virtual-agent/tests/Chatbox.unit.spec.js [hunk 2]
```diff
@@ -69,6 +69,17 @@ describe.skip('App', () => {
     );
   }
 
+  async function wait(timeout) {
+    return new Promise(resolve => {
+      setTimeout(resolve, timeout);
```
Comment (ksk385) [inline src/applications/virtual-agent/tests/Chatbox.unit.spec.js:74]: Should we be using timeouts in the tests now?

Comment (eubiera) [reply src/applications/virtual-agent/tests/Chatbox.unit.spec.js:74]: It's a different usage in this case. The actual elapse of time is all timestamp based, using normal JS date functions.  
In this case it's getting called for a tenth of a second (not an hour). It seems the breaking changes were a result of using useFakeTimers, and the tick() call was not working for some reason.

File: src/applications/virtual-agent/components/chatbox/Chatbox.jsx (hunk 1 above)
Comment (eubiera) [reply src/applications/virtual-agent/components/chatbox/Chatbox.jsx:18]: We'll make a point of removing this in the future.

File: src/applications/virtual-agent/tests/Chatbox.unit.spec.js (hunk 2 above)
Comment (eubiera) [reply src/applications/virtual-agent/tests/Chatbox.unit.spec.js:74]: It's a different usage in this case. The actual elapse of time is all timestamp based, using normal JS date functions.
In this case it's getting called for a tenth of a second (not an hour). It seems the breaking changes were a result of using useFakeTimers, and the tick() call was not working for some reason.

File: src/applications/virtual-agent/components/chatbox/Chatbox.jsx (hunk 1 above)
Comment (eubiera) [reply src/applications/virtual-agent/components/chatbox/Chatbox.jsx:18]: We'll make a point of removing this for sure.

**Pull request merged**